src/ray/core_worker/actor_manager.cc

```diff
@@ -70,7 +70,7 @@ bool ActorManager::AddNewActorHandle(std::unique_ptr<ActorHandle> actor_handle,
                         caller_address, actor_id, actor_creation_return_id);
 }
 
-bool ActorManager::AddActorHandle(std::unique_ptr<ActorHandle> actor_handle,
+bool ActorManager::AddActorHandle(std::unique_ptr<core::ActorHandle> actor_handle,
```

Comment:
I think adding `using namespace ray::core;` in .cc files is much simpler. We may move more classes from `ray` to `ray::core` later.